packages/captable/README.md

```diff
@@ -1,4 +1,55 @@
 # Smart contracts for CapTable BRØK
 
-NB! brokProd is still against Arbitrum Goerli
+Smart contracts for CapTable and CapTableRegistry developed in [Solidity v0.8.0](https://docs.soliditylang.org/en/v0.8.0/)
 
+Based on [ERC1400](https://www.shipfinex.com/blog/erc-1400-a-standardized-framework-for-security-tokens), but intended to migrate to [ERC3643](https://www.erc3643.org/) in the future.
+
+The contracts are built and tested with [Hardhat](https://hardhat.org/hardhat-runner/docs/getting-started#overview)
+
+Based on the generated contracts, [tsup](https://www.npmjs.com/package/tsup) is used to generate a TypeScript library, to be used by Brøk API and The Graph configuration.
+
+***NB! brokProd is still against Arbitrum Goerli***
+
+## Project overview
+
+| **Folder** | **Description** |
+|--|--|
+| `artifacts/` | Compiled artifacts generated by Hardhat compile (generated) |
+| `cache/` | The directory used by Hardhat to cache its internal stuff (generated) |
+| `contracts/` | CapTable and CapTableRegistry contracts in Solidity, with dependencies |
+| `deployments/` | Our way to persist smart contract addresses deployed on Arbitrum, generated by [./tasks/generate-deployments.ts](./tasks/generate-deployments.ts) |
+| `dist/` | Generated TypeScript library of CapTable and CapTableRegistry (generated) |
+| `scripts/` | I dont know what this does |
+| `tasks/` | Hardhat tasks to be run with ``hardhat <task name> --network <network name> --log``
+| `test/` | Tests for CapTable and CapTableRegistry |
+| `typechain-types/` | Generated TypeScript bindings for smart contracts (generated) |
+
+
+### .env
+| **Value** | **Description** |
+|--|--|
+| `ETHERSCAN_API_KEY` | API key to Etherscan, used by/in ??? @jon |
+| `SEED_DEV` | Mnemonic key for Brønnøysundregistrenes wallet used to deploy contracts in the dev environment |
+| `SEED_STAGE` | Mnemonic key for Brønnøysundregistrenes wallet used to deploy contracts in the stage environment |
+| `SEED_PROD` | Mnemonic key for Brønnøysundregistrenes wallet used to deploy contracts in the production environment **NB: contains real money** |
+| `RPC_CONTAINER` | Network address to Hardhat network running with `docker-compose` |
+| `RPC_LOCAL` | Network address to Hardhat network running local |
+| `RPC_TESTNET` | Network address to the Arbitrum Goerli network |
+| `RPC_STAGENET` | Network address to the Arbitrum One network |
+| `RPC_MAINNET` | Network address to the Arbitrum One network |
+| `REPORT_GAS` | ??? @jon |
```

Comment:
Same as CONTRACT_SIZER but with finer granularity output. This outputs gas used for each test and every smart contract method

Reply:
lagt inn forslaget